src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/models/BuzzModel.java

```diff
@@ -0,0 +1,34 @@
+package com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.impl.models;
+
+public class BuzzModel extends Model implements com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.models.Model {
+
+    public final static String BUZZ = "Buzz";
```

Comment:
This really should be a configurable property read in at startup-time to ensure i18n